cbits/is-valid-utf8.c

```diff
@@ -331,10 +335,26 @@ static int8_t const ef_fe_lookup[16] = {
 };
 
 __attribute__((target("ssse3"))) static inline bool
-is_ascii_sse2(__m128i const *src) {
+is_ascii_sse2(__m128i const *src, __m128i const prev_first_len) {
```

Comment:
I'm not sure why that happened actually. I suspect it's because I thought SSE2-only could make use of such a function, but in the end it never did, and the SSE-optimized UTF-8 checker needs SSSE3 at least.